Src/StackifyLib/Internal/Auth/Claims/AppClaimsManager.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Threading.Tasks;
+
+namespace StackifyLib.Internal.Auth.Claims
+{
+    internal static class AppClaimsManager
+    {
+        private readonly static IAppClaimsBuilder _claimsBuilder = AppClaimsBuilderFactory.Get();
+
+        private static AppClaims _claims = null;
+
+        public static AppClaims Get()
+        {
+            if (_claims != null)
+                return _claims;
+
+            GetAsync().Wait();
```

Comment:
ideally, we'd like to see that background thread return i think? i'd need to look a bit deeper into the timer to see if we gain anything. But, I think we'd like to keep everything downstream async and if we _have_ to only hang in one place